src/cli/func-api.ts

```diff
@@ -16,6 +16,36 @@ export class FuncAPI {
     return funcCliCommand(['version']);
   }
 
+  static invokeFunctionLocal(
+    id: string,
+    location: string,
+    contextType: string,
+    format: string,
+    source: string,
+    type: string,
+    data: string,
+  ): string {
+    if (id) {
+      return `func invoke --id ${id} -p ${location} --content-type ${contextType} -f ${format} --source ${source} --type ${type} --data ${data}`;
+    }
+    return `func invoke -p ${location} --content-type ${contextType} -f ${format} --source ${source} --type ${type} --data ${data}`;
+  }
+
+  static invokeFunctionRemote(
+    id: string,
+    namespacs: string,
+    contextType: string,
+    format: string,
+    source: string,
+    type: string,
+    data: string,
+  ): string {
+    if (id) {
+      return `func invoke --id ${id} -n ${namespacs} --content-type ${contextType} -f ${format} --source ${source} --type ${type} --data ${data}`;
+    }
+    return `func invoke -n ${namespacs} --content-type ${contextType} -f ${format} --source ${source} --type ${type} --data ${data}`;
```

Comment:
When trying to execute the invoke action i get 
![image](https://user-images.githubusercontent.com/49404737/160754693-b466c95e-f644-4400-b687-4c08489c22ba.png)